src/Configuration.h

```diff
@@ -46,6 +46,8 @@ class Configuration
   byte findExistingEvent(unsigned int nn, unsigned int en) const;
   byte findEventSpace() const;
   byte findExistingEventByEv(byte evnum, byte evval) const;
+  
+  unsigned int getFreeEEbase();
```

Comment:
Do we need a getter function? We don't have one for any other EE_ variable.

Reply:
No, we don't need the getter function.  It compiles fine without it.